### ISSUE TITLE
Validation error

### ISSUE THREAD

**Issue by simoncarre** [akeneo-labs/CustomEntityBundle#124]
Hi!

I'm using the bundle with simple custom entity `Brand` (like in the example).
I added `validation.yml` file with this content in this directory `Acme/Bundle/MyBundle/Resources/config/` :

``` yml
Acme\Bundle\MyBundle\Entity\Brand:
    properties:
        label:
            - NotBlank: ~
        code:
            - NotBlank: ~
```

But saving an entity with an empty field generates an error in the logs and the error is not raised on front:

```
request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\BadRequestHttpException: "Invalid data" at /var/www/steam-pim/src/vendor/akeneo-labs/custom-entity-bundle/Action/Rest/UpdateAction.php line 54 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\BadRequestHttpException(code: 0): Invalid data at /var/www/steam-pim/src/vendor/akeneo-labs/custom-entity-bundle/Action/Rest/UpdateAction.php:54)"}
```
How can add clean validation on my entity?

Thank you!
Simon

**Comment by jmleroux** [comment]
Hello @simoncarre ,
I don't really understand the question. If a value is blank, the error is normal. Is your problem that there is no error on front?

**Comment by simoncarre** [comment]
Yes but I have no error on front. Just error on async request and the screen is lock on loader icon.
Do you have solution for that?

Thank you!

**Comment by jmleroux** [comment]
Which version do you use?

**Comment by simoncarre** [comment]
2.0.6 for Akeneo PIM and Custom Entity Bundle.

**Comment by kevinrademan** [comment]
I'm also interested in this. I have the same behavior using akeneo 2.0.11 and using 2.0.8 of the custom entity module. 

I did some brief investigation into this problem and it appears that the validation errors are available within the controller, but they are not being passed back in the error response. 
https://github.com/akeneo-labs/CustomEntityBundle/blob/881aa6bf7a9dd8ab20669cb7f9a2fe679575ba35/Action/Rest/CreateAction.php#L53
https://github.com/akeneo-labs/CustomEntityBundle/blob/1dbdebb7227e1c3731dd76bea4d5783f49b8fc95/Action/Rest/UpdateAction.php#L54

Assuming we pass these errors back in the response, how would we link those to validation errors shown in the UI?

**Comment by simoncarre** [comment]
Hi @jmleroux!

Do you know on which release the fix will be included?

Thank you in advance.

**Comment by mmetayer** [comment]
Hi @simoncarre, the fix will be included in the next version (2.1.3)
